lib/qx/tool/cli/commands/contrib/Install.js

```diff
@@ -88,28 +88,31 @@ qx.Class.define("qx.tool.cli.commands.contrib.Install", {
         repos_cache = this.getCache().repos;
       }
 
-      if (!argv.repository) {
+      if (!argv.contrib_uri) {
         await this.__loadFromContribJson();
         return;
       }
    
       let qooxdoo_version = await this.getUserQxVersion();
       if(argv.verbose) console.info(`>>> qooxdoo version:  ${qooxdoo_version}`);
      
-      // has a repository name been given?
-      if (!argv.repository) {
-        throw new qx.tool.cli.Utils.UserError("No repo name given");
+      // has a contrib_uri name been given?
+      if (!argv.contrib_uri) {
+        throw new qx.tool.cli.Utils.UserError("No contrib resource identifier given");
       }
-      let repo_name = argv.repository;
+      // currently, the contrib uri is github_username/repo_name[/path/to/repo].
```

Comment:
this should be `/path/to/dir-containing-manifest` not `path/to/repo/`, but I can correct that later...